src/pathOrderOptimizer.cpp

```diff
@@ -199,7 +199,7 @@ int PathOrderOptimizer::getRandomPointInPolygon(int poly_idx)
 
 static inline bool pointsAreCoincident(const Point& a, const Point& b)
 {
-    return vSize2(a - b) < 25; // points are closer than 5uM, consider them coincident
+    return vSize2(a - b) < 100; // points are closer than 10uM, consider them coincident
```

Comment:
We think this is a good change, but could you instead use the setting `meshfix_maximum_resolution` for this? It acts effectively as the slicing resolution, so we think it would be fitting to have that instead of hard-coding this value.

Reply:
That would be sensible. The only difficulty is that setting is not currently accessible within the LineOrderOptimizer class so it would require passing in via the constructor. Is that what you would like me to do?

Reply:
Actually, it gets worse because where the LineOrderOptimizer is used in LayerPlan::addLinesByOptimzer(), the mesh isn't available either so it would have to be passed in to that method from wherever it is called from. All getting a bit messy. How should we proceed?